src/07-embedding/01-minimal.ts

```diff
@@ -0,0 +1,133 @@
+import OpenAI from "openai"
+import dotenv from "dotenv"
+import weaviate from "weaviate-ts-client"
+import _ from "lodash"
+
+dotenv.config()
+const openai = new OpenAI()
+
+// spell-checker:disable
```

Comment:
TIL these spell-checker [hints](https://github.com/streetsidesoftware/vscode-spell-checker?tab=readme-ov-file#in-document-settings).